packages/graphile-utils/src/makeChangeNullabilityPlugin.ts

```diff
@@ -1,49 +1,121 @@
 import type {
-  SchemaBuilder,
   Build,
   Context,
-  Plugin,
   Options,
+  Plugin,
+  SchemaBuilder,
 } from "graphile-build";
-import type { GraphQLInputFieldConfig, GraphQLFieldConfig } from "graphql";
+import {
+  GraphQLFieldConfig,
+  GraphQLInputFieldConfig,
+  GraphQLInputType,
+  GraphQLNonNull,
+  GraphQLType,
+} from "graphql";
+import { GraphQLList } from "graphql/type/definition";
```

Comment:
You must only import *types* here, the underlying `GraphQLList` / `GraphQLNonNull` must come from `build.graphql.GraphQLList`/etc.

Note I've showing `import type` here but that syntax might not be supported in the current version of TypeScript - either upgrade TypeScript or just know to only import types here.

```suggestion
import type {
  GraphQLFieldConfig,
  GraphQLInputFieldConfig,
} from "graphql";
```

Reply:
✅